cmd/mount.go

```diff
@@ -434,8 +437,8 @@ func NewReloadableStorage(format *meta.Format, cli meta.Meta, patch func(*meta.F
 			patch(new)
 		}
 		old := &holder.fmt
-		if new.Storage != old.Storage || new.Bucket != old.Bucket || new.AccessKey != old.AccessKey || new.SecretKey != old.SecretKey || new.SessionToken != old.SessionToken {
-			logger.Infof("found new configuration: storage=%s bucket=%s ak=%s", new.Storage, new.Bucket, new.AccessKey)
+		if new.Storage != old.Storage || new.Bucket != old.Bucket || new.AccessKey != old.AccessKey || new.SecretKey != old.SecretKey || new.SessionToken != old.SessionToken || new.StorageClass != old.StorageClass {
```

Comment:
The storage class is specified by current client, it should not overwritten by reload

Reply:
```
cli.OnReload(func(new *meta.Format) {
    if patch != nil {
	    patch(new)
    }
```
The priority of 'mount --storage-class' is higher than that configured for storage in meta. Even if you change the configuration in meta, it will not be reloaded when `mount --storage-class xxx`. 
Just like the 'mount --bucket' parameter.